CodeSignalJS/areSimilar.ts

```diff
@@ -8,7 +8,22 @@
 // Ex input: a = [1, 2, 3] and b = [3, 1, 2] => false
 
 function areSimilar(a: any[], b: any[]): boolean {
-  return true;
+  let swapCount = 0;
+
+  for (let i = 0; i < a.length; i++) {
+    // check equality of each element 
+    if (a[i] !== b[i]) {
+      // check for swap
+      if (a[i] === b[i + 1]) {
```

Comment:
I should have built better test cases for this problem 😅 
This solution is perfect so long as the swaps must be sequential elements, however if I were to add the test case

```
test('Case: 6', () => {
    expect(func([1, 2, 3, 4], [4, 2, 3], 1)).toBe(true);
  });
```

which means that swapping can be with any indices, how might you change your solution?